pkg/util/ipvs/ipvs.go

```diff
@@ -0,0 +1,131 @@
+/*
+Copyright 2014 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package ipvs
+
+import (
+	"net"
+	"sync"
+
+	godbus "github.com/godbus/dbus"
+	utildbus "k8s.io/kubernetes/pkg/util/dbus"
+	utilexec "k8s.io/kubernetes/pkg/util/exec"
+	utilsysctl "k8s.io/kubernetes/pkg/util/sysctl"
+)
+
+const (
+	DefaultIPVSScheduler = "rr"
+
+	ipvsSvcFlagPersist   = 0x1
```

Comment:
You define these here, and then use them in exactly one place - to initialize more consts.  Better to just initialize below from the literals